cmd/e2e-cluster/commands/setup.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/grafana/xk6-disruptor/pkg/testutils/e2e/cluster"
+	"github.com/spf13/cobra"
+)
+
+// BuildSetupCmd returns the setup command
+func BuildSetupCmd() *cobra.Command {
+	name := cluster.DefaultE2eClusterConfig().Name
+	port := cluster.DefaultE2eClusterConfig().IngressPort
+	var images []string
+
+	cmd := &cobra.Command{
+		Use:   "setup",
+		Short: "creates and configures an e2e test cluster ",
+		Long:  "creates and configures an e2e test cluster with default options.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cluster, err := cluster.BuildE2eCluster(
+				cluster.DefaultE2eClusterConfig(),
+				cluster.WithEnvOverride(false),
+				cluster.WithName(name),
+				cluster.WithIngressPort(port),
+				cluster.WithImages(images...),
+			)
+			if err != nil {
+				return fmt.Errorf("failed to create cluster: %w", err)
+			}
+
+			//nolint:forbidigo // allow printf as this is a CLI tool
+			fmt.Printf("cluster %q created\n", cluster.Name())
```

Comment:
Perhaps an alternative could be using `log` for this message, as it indeed seems like an informative log message.

Reply:
The main reason for not doing so is to be able to capture the name of the cluster and pass it as an env variable to the tests.  Formating this kind of output as a log entry is in my opinion abuse of the idea of a log.

That said, I think adding logging and an option for controlling the verbosity would be highly beneficial, but scapes the scope of this PR.

Reply:
Oh, I did not realize this output was parsed. It makes sense not using a log then!